src/devices/bus/a2bus/cards.cpp

```diff
@@ -68,6 +68,8 @@
 #include "bus/a2bus/lancegs.h"
 #include "bus/a2bus/titan3plus2.h"
 #include "bus/a2bus/softcard3.h"
+#include "bus/a2bus/a2superdrive.h"
```

Comment:
Please put this in alphabetical order in the headers here to keep in manageable.